ocrd_monitor/README.md

```diff
@@ -0,0 +1,60 @@
+# OCRD-Monitor
+
+The OCRD-Monitor application is used to monitor the progress and results of OCR-D jobs.
+In order to run the application you need to have an OCR-D Controller as well as the Dozzle container for viewing logs set up.
+To launch the monitor, first create a script containing the following code:
+
+```bash
+export OCRD_BROWSER__MODE=<native or docker>
+export OCRD_BROWSER__WORKSPACE_DIR=<dir-containing-ocrd-workspaces>
+export OCRD_BROWSER__PORT_RANGE="[9000,9100]"
+export OCRD_CONTROLLER__JOB_DIR=<dir-containing-ocrd-job-files>
+export OCRD_CONTROLLER__HOST=<url-to-ocrd-controller>
+export OCRD_CONTROLLER__PORT=<port-of-ocrd-controller>
+export OCRD_CONTROLLER__USER=<name-of-user-on-ocrd-controller>
+export OCRD_CONTROLLER__KEYFILE=<path-to-private-ssh-key>
+export OCRD_LOGVIEW__PORT=<port-of-dozzle>
+
+cd ocrd_monitor
+uvicorn "ocrdmonitor.main:app" --reload
+```
+
+Make sure to install the requirements from `requirements.txt`:
+
+```bash
+pip install -r requirements.txt
+```
+
+Finally launch the application using the script you created.
+The OCRD-Monitor will be available at `http://localhost:8000`
+
+
+## Testing
+
+1. Install runtime and dev dependencies 
+```bash
+    pip install -r requirements.txt
+    pip install -r requirements.dev.txt
+```
+
+2. Run nox or pytest
+```bash
+    nox
+```
+
+```bash
+    pytest tests
+```
```

Comment:
The problem here is that we cannot do this when running our regular containers. (Or should we `ADD` the tests to the image as well?)

The only way I could run this is by copying to the /data volume and running from there...

Reply:
The tests do not require any real ocrd results or workspaces, so you shouldn't need to mount a `/data` volume.
There are a few test cases that require docker containers, though (logging into an SSH server and querying process data), so these won't work inside a container.

Reply:
The tests need `./tests` (which in turn need to have ocrdmonitor and ocrdbrowser installed locally or as packages). So copying everything to /data volume and running from there is the only way to get it running in the container setup.

But of course we could argue here that this is not an integration test, and these are to be run in the git source without Docker. (But the docs should be more explicit IMO. Perhaps even provide a Makefile which encapsulates dependencies and tests on the one hand, and Docker build and run on the other?)